platform/remarkable/koreader.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/sh
 export LC_ALL="en_US.UTF-8"
-
 # working directory of koreader
 KOREADER_DIR="${0%/*}"
 
 # we're always starting from our working directory
 cd "${KOREADER_DIR}" || exit
 
+# reMarkable 2 check
+if [ "reMarkable 2.0" == "$(</sys/devices/soc0/machine)" ]; then
```

Comment:
Does the rM ship with Bash?

I'm not quite sure bb ash supports the io slurp operator like that...

(Also, the posix shell equality test is a *single* equal sign, much to my dismay).

Reply:
Which leaves the `read` builtin, we use constructs like

`IFS= read -r ORIG_FB_ROTA <"/sys/class/graphics/fb0/rotate"` elsewhere, for instance.

Reply:
That's exactly why I said to leave this mostly unrelated change to another PR, for future reference. Now this can't be merged like it could yesterday. ;-)

Reply:
the read doesn't bother me. the rm has bash.

Reply:
Shellcheck would presumably complain that you're using a Bashism without using the correct shebang and so will I. ;-)

My preferred solution as a rule of thumb is not to use Bashisms unless it's really awkward not to.

Reply:
PS Please allow CircleCI access so it'll tell you this automatically.

Reply:
how do i give circleci access?

Reply:
It used to just run on PRs, but for some reason it looks like people now have to go to https://circleci.com/ and sign up with their GitHub account.

It's very annoying too, because the CI is probably at its most useful with new contributors (not talking about you ;-) ).

But I'll have to check if maybe some config changed on their or GH's side too.

Reply:
it had failed on circleci with unauthorized